_config.yml

```diff
@@ -115,7 +115,8 @@ defaults:
 include:
   # dotfiles are excluded by default
   - .well-known
-  - docs/accessibility-assessment-ial1-account-creation.pdf
```

Comment:
we need to add a redirect to catch because the old URLs/sitemap check is failing on it no longer existing

Reply:
I believe the redirects are there to satisfy the tests, rather than redirect. They don't work.

Reply:
If I recall, we don't have the ability to redirect PDFs.

Reply:
ah dang that's right, I forget what we did for this last time, I'll try to dig it up